apps/arena/lib/arena/entities.ex

```diff
@@ -39,8 +41,9 @@ defmodule Arena.Entities do
         power_up_damage_modifier: config.power_ups.power_up.power_up_damage_modifier,
         inventory: nil,
         damage_immunity: false,
-        effects: %{},
-        cooldowns: %{}
+        effects: [],
+        cooldowns: %{},
+        bonus_damage: 0
```

Comment:
just to comment this but this variable sounds like an additive modifier, maybe `bonus_damage_multiplier` would be more suitable